Workbooks/AcscEssential8.json

```diff
@@ -3474,7 +3583,7 @@
       }
     }
   ],
-  "fallbackResourceIds": [],  
-  "fromTemplateId": "sentinel-AcscEssential8Workbook",
+  "fallbackResourceIds": [],
+  "fromTemplateId": "sentinel-AcscEssential8",
```

Comment:
@nickt444, maybe the templateid is changed, can you please revert the change, thanks.

Reply:
done